src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
+
+    <conversionRule conversionWord="highlightex" converterClass="org.example.server.LogbackHighlightConfig"/>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss} %highlightex([%level]): %highlightex(%msg%n)</pattern>
```

Comment:
Time format should be `h:mm:ss a` to match what it was before (I use 24 hour time myself but I assumed there would be more people wanting 12 hour time)